colander/tests/test_colander.py

```diff
@@ -1557,6 +1557,13 @@ def test_serialize_string_with_high_unresolveable_high_order_chars(self):
         e = invalid_exc(typ.serialize, node, not_utf8)
         self.assertTrue('cannot be serialized' in e.msg)
 
+    def test_serialize_encoding_with_non_string_type(self):
+        utf8 = '123'.encode('utf-8')
```

Comment:
Or just b'123', in fact (ASCII and UTF-8 are identical encodings for 7-bit values).